tests/vse_sync_pp/parsers/test_phc2sys.py

```diff
@@ -0,0 +1,46 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Test cases for vse_sync_pp.parsers.phc2sys"""
+
+from unittest import TestCase
+from decimal import Decimal
+
+from vse_sync_pp.parsers.phc2sys import (
+    TimeErrorParser,
+)
+
+from .test_parser import ParserTestBuilder
+
+class TestTimeErrorParser(TestCase, metaclass=ParserTestBuilder):
+    """Test cases for vse_sync_pp.parsers.phc2sys.TimeErrorParser"""
+    constructor = TimeErrorParser
+    id_ = 'phc2sys/time-error'
+    elems = ('timestamp', 'terror', 'state', 'delay')
+    accept = (
+        (   'phc2sys[681011.839]: [ptp4l.0.config] '
```

Comment:
Also need a case where ` [ptp4l.0.config]` (including leading single space) is absent